cmd/gateway-gcs.go

```diff
@@ -31,6 +31,8 @@ import (
 	"strings"
 
 	"cloud.google.com/go/storage"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/cloudresourcemanager/v1"
```

Comment:
> Is this vendorized properly?

Not sure yet, make pkg-add doesn't automatically add dependencies, I'll wait until this test shows some errors if something is missing.